tasks/bump.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// takes care of bumping the version number in package.json
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('bump', {
+    options: {
+      files: ['package.json', 'npm-shrinkwrap.json'],
+      bumpVersion: true,
+      commit: true,
+      commitMessage: 'Release v%VERSION%',
+      commitFiles: ['package.json', 'npm-shrinkwrap.json', 'CHANGELOG'],
+      createTag: true,
+      tagName: 'v%VERSION%',
+      tagMessage: 'Version %VERSION%',
+      push: false,
```

Comment:
Thanks, @rishibaldawa 

I'll note one thing about this change. At present, the tags have been of the form 'L.M.N', which is different from fxa-{content,auth,customs,auth-db,etc.}-server' repositories tagging. I'd been meaning to ask @seanmonstar if he was okay with that change. But the inconsistency of the tagging pattern has been something that is just a nuisance in deployments.

Otherwise, LGTM!